index.ts

```diff
@@ -49,10 +49,8 @@ console.log(`Update count: ${state.updates}`);
  */
 
 // A Membrane program can export _actions_ that you invoke. This program exports a `run` action.
-// Let's invoke the `run` action, defined as a function below. To invoke an action, you can:
-// 1. Click on the "Invoke ▶️" button directly above the function signature
-// 2. In the Membrane Explorer, click "getting-started" then click "Invoke" to the right of "run"
-// 3. Use the Command+Enter keyboard shortcut
```

Comment:
I think it's okay to remove this for now but we should fix it soon

Reply:
[T-839](https://membrane.height.app/T-839) Add back the Invoke ▶️ button in the IDE